blockbase/inc/social-navigation.php

```diff
@@ -4,9 +4,7 @@
 // to social links in the following conditions.
 function blockbase_condition_to_render_social_menu( $block_content, $block ) {
 	// The block should be a navigation block.
-	if ( 'core/navigation' !== $block['blockName'] ) {
-		return false;
-	}
+	if ( 'core/navigation' !== $block['blockName']){  return false;}
```

Comment:
I tried lots of things but I can't get it to fix this!

Reply:
Oh it doesn't fix that for me either! I was testing in an Archeo pattern, it seemed to pick up on me removing spaces between brackets in those files. I wonder what's different about this file..

Reply:
Weird, I just made some changes in this file to test and phpcs picked them up + fixed them: https://github.com/Automattic/themes/pull/5493/commits/caa8a5edcc67fedd47b86b697b0d0ffced0e866c